bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/PersistenceFilter.java

```diff
@@ -17,6 +17,6 @@
  *
  * @author Markus Rathgeb - Initial contribution
  */
-public class SimpleFilter {
+public class PersistenceFilter {
```

Comment:
Should we add nullness annotations and other touched interfaces / classes?

Reply:
Added.